src/main/scala/org/constellation/genesis/GenesisObservationWriterException.scala

```diff
@@ -0,0 +1,13 @@
+package org.constellation.genesis
+
+sealed trait GenesisObservationWriterException extends Throwable {
+  val exceptionMessage: String
+}
+
+case class CannotWriteGenesisObservationToDisk(exception: Throwable) extends GenesisObservationWriterException {
```

Comment:
If you need a message only, it's better to use companion objects as follows:
```
case class CannotWriteGenesisObservationToDisk(message: String) extends .. {
 override val exceptionMessage = message
}
object CannotWriteGenesisObservationToDisk {
 def apply(exception: Throwable) = new CannotWriteGenesisObservationToDisk(exception.getMessage)
}
```
That way, you'll not need to store the whole exception in the case class (which takes a memory) but only the string message. You can take a look how CheckpointBlockValidator is created